[docs] EAS Update: add more links to api (#23096)
expo/expo: docs/pages/eas-update/faq.mdx

description: Frequently asked questions about EAS Update.
---

### How can I implement a custom update strategy for my app?

> The default update strategy for `expo-updates` is to check for updates every time the app is loaded. You can implement a custom update strategy with the [Updates API](/versions/latest/sdk/updates) and [app.json configuration](/versions/latest/config/app/#updates).

Reviewer: ```suggestion
By default, `expo-updates` checks for updates every time the app is loaded. You can implement a custom update strategy with the [Updates API](/versions/latest/sdk/updates) and [app config configuration](/versions/latest/config/app/#updates).
```

I think the start of the sentence can be concise. Also, instead of using `app.json`, let's use app config since "app config" can also mean `app.config.js` or other variants of the config file.
